lighthouse-core/lib/emulation.js

```diff
@@ -16,15 +16,56 @@ const NO_CPU_THROTTLE_METRICS = {
   rate: 1,
 };
 
+/**
+ * @param {string} userAgent
+ * @param {LH.Config.Settings['formFactor']} formFactor
+ * @return {LH.Crdp.Emulation.SetUserAgentOverrideRequest['userAgentMetadata']}
+ */
+function parseUseragentIntoMetadata(userAgent, formFactor) {
+  const match = userAgent.match(/Chrome\/([\d.]+)/); // eg 'Chrome/(71.0.3577.0)'
+  const fullVersion = (match && match[1]) || '99.0.1234.0';
+  const version = fullVersion.split('.', 1)[0];
+  const brands = [
+    {brand: 'Chromium', version},
+    {brand: 'Google Chrome', version},
```

Comment:
wdyt of adding a entry for LH version here?

Reply:
and if we do that, we should randomly sort the array, to support the GREASE stuffs (or will chrome do that for us in the request headers?)

Reply:
> wdyt of adding a entry for LH version here?

done

> and if we do that, we should randomly sort the array, to support the GREASE stuffs (or will chrome do that for us in the request headers?)

seems like fun.. and its jsut adding a `.sort(_ => Math.random() - 0.5)`  

... but we add some non-determinimism without much gain so i think we should put it off. leave the greasing to the big boiz.